.buildkite/pipeline-release-redhat.yml

```diff
@@ -1,50 +1,61 @@
-          - "build-operatorhub-tool"
-        commands:
-          - make generate-crds-v1
-        artifact_paths:
-        - "config/*.yaml"
-
-      - label: "run container push"
-        key: "redhat-container-push"
-        depends_on:
-          - "generate-crds"
-          - "build-operatorhub-tool"
-        commands:
-          - buildkite-agent artifact download "bin/operatorhub" /usr/local/
-          - chmod u+x /usr/local/bin/operatorhub
-          - cd hack/operatorhub
-          - operatorhub container push --dry-run=\${DRY_RUN:-true}
-
-      - label: "run container preflight"
-        key: "redhat-preflight"
-        if: build.env("DRY_RUN") == "false"
-        depends_on:
-          - "redhat-container-push"
-        commands:
-          - .buildkite/scripts/release/operatorhub.sh preflight
-
-      - label: "run remaining release"
-        key: "redhat-release"
-        depends_on:
-          - "build-operatorhub-tool"
-          - "redhat-container-push"
-          - "redhat-preflight"
-        commands:
-          - buildkite-agent artifact download "config/*.yaml" .
-          - buildkite-agent artifact download "bin/operatorhub" /usr/local/
-          - chmod u+x /usr/local/bin/operatorhub
-          - .buildkite/scripts/release/operatorhub.sh release
\ No newline at end of file
+
+  - label: ":go: build operatorhub tool"
+    key: "build-operatorhub-tool"
+    commands:
+      - cd hack/operatorhub
+      - make build
+      - buildkite-agent artifact upload bin/operatorhub
+
+  - label: ":docker: push container"
+    key: "redhat-container-push"
+    depends_on:
+      - "build-operatorhub-tool"
+    commands:
+      - buildkite-agent artifact download "bin/operatorhub" /usr/local/ && chmod u+x /usr/local/bin/operatorhub
+      - |
+        cd hack/operatorhub
+        operatorhub container push
+
+  - label: ":docker: preflight container check"
+    key: "redhat-preflight"
+    if: build.env("OHUB_DISABLE_PREFLIGHT") != "true"
+    depends_on:
+      - "build-operatorhub-tool"
+      - "redhat-container-push"
+    commands:
+      - .buildkite/scripts/release/redhat-preflight.sh $$BUILDKITE_TAG
+
+  - label: ":docker: publish container"
+    key: "redhat-publish"
+    depends_on:
+      - "build-operatorhub-tool"
+      - "redhat-container-push"
+      - "redhat-preflight"
+    commands:
+      - buildkite-agent artifact download "bin/operatorhub" /usr/local/ && chmod u+x /usr/local/bin/operatorhub
+      - |
+        cd hack/operatorhub
+        operatorhub container publish
+
+  - label: ":redhat: generate and create-pr"
+    key: "redhat-release"
+    depends_on:
+      - "build-operatorhub-tool"
+      - "redhat-publish"
+    commands:
+      - buildkite-agent artifact download "bin/operatorhub" /usr/local/ && chmod u+x /usr/local/bin/operatorhub
+      - |
+        cd hack/operatorhub
+        #operatorhub generate-manifests --yaml-manifest=../../config/crds.yaml --yaml-manifest=../../config/operator.yaml
```

Comment:
I suspect you intended to remove this.

Reply:
Yes, it seems to me more correct to download the released manifests rather than to regenerate them but this has the downside that we can't run the tool (even with `--dry-run=true`) on a version for which we have not yet published manifests.